EN: a few client orders and tutorial windows

allister-translator opened the pull request:
hey i'm totally new to this github thing i hope i'm doing this right (´・ω・`)

allister-translator committed:
Update co_011190.csv

allister-translator committed:
Update co_011190.csv

allister-translator committed:
Update co_050025.csv

allister-translator committed:
Update wh_storyboard_questrank.csv

alama commented on Files/co_050025.csv:
```diff
@@ -1,7 +1,7 @@
+text04#0,"""First, please meet with me.<br>I shall be waiting at the quest counter<br>in the Gate area."""
 text100#0,"""『クライアントオーダー』について"""
-text101#0,"""コフィーにクライアントオーダーが発生しました。<br><br>コフィーは<br>ゲートエリア・東クエストカウンターにいます。"""
+text101#0,"""A client order from Koffie is available.<br><br>Koffie is locatedd at the<br>quest counter in the Gate Area."""
```
You need to move this file OUT of the `Files` folder

alama commented on Files/wh_storyboard_questrank.csv:
```diff
@@ -1,2 +1,2 @@
-name01#0,"""ストーリークエストの難易度について"""
-whelp_storyboard_questrank01#0,"""<yellow>【ストーリークエストの難易度について】<c><br><br>\u3000ストーリークエストの難易度を選ぶことができます。<br>\u3000どちらを選んでも、<yellow>ストーリーに変化はありません。<c><br><br>\u3000<c 00ffff>「イージー」<c><br>\u3000\u3000エネミーのレベルが低く、楽にプレイできます。<br>\u3000\u3000ストーリーの続きが気になる場合におすすめです。<br><br>\u3000<c 00ffff>「ノーマル」<c><br>\u3000\u3000エネミーのレベルが高く、ボスも強くなります。<br>\u3000\u3000クリアが難しくなりますが、一部の称号を獲得できます。"""
+name01#0,"""Story Quest Difficulty"""
+whelp_storyboard_questrank01#0,"""<yellow>【Story Quest Difficulty】<c><br><br>\u3000You can select a difficulty for Story Quests.<br>\u3000Regardless of what difficulty you choose,<yellow>the story does not change.<c><br><br>\u3000<c 00ffff>「Easy」<c><br>\u3000\u3000Enemy levels are lower, allowing you to enjoy the story.<br>\u3000\u3000If you're curious about the next<br><br>\u3000<c 00ffff>「Norma ...
```
You need to move this file OUT of the `Files` folder as well

allister-translator committed:
Rename Files/wh_storyboard_questrank.csv to Tutorial/Window/wh_storyboard_questrank.csv

allister-translator committed:
Rename Files/co_050025.csv to Tutorial/Dialogue/co_050025.csv

alama commented on Tutorial/Dialogue/co_050025.csv:
```diff
@@ -0,0 +1,7 @@
+text03#0,"""Your recent activity log suggests you're<br>eligible to explore locations deeper in the Forest area.<br>I'll guide you through the authorization process."""
+text04#0,"""First, please meet with me.<br>I shall be waiting at the quest counter<br>in the Gate area."""
+text100#0,"""『クライアントオーダー』について"""
+text101#0,"""A client order from Koffie is available.<br><br>Koffie is locatedd at the<br>quest counter in the Gate Area."""
```
Ok, now the CircleCI bot say "locatedd" is a spelling mistake

allister-translator committed:
Update co_050025.csv

alama commented on Tutorial/Dialogue/co_050025.csv:
```diff
@@ -3,5 +3,5 @@ name01#0,"""Koffie"""
 text04#0,"""First, please meet with me.<br>I shall be waiting at the quest counter<br>in the Gate area."""
-text100#0,"""『クライアントオーダー』について"""
-text101#0,"""A client order from Koffie is available.<br><br>Koffie is locatedd at the<br>quest counter in the Gate Area."""
+text100#0,"""Regarding "Client Orders""""
```
You added an extra `"` at the end of this line

allister-translator committed:
Update co01042100.csv

allister-translator committed:
Update co01042101.csv

alama committed:
Update co_050025.csv

alama committed:
Update co_050025.csv

https://github.com/Arks-Layer/PSO2ENPatchCSV/wiki#user-content-reminders-

You have to use `\""` for quotes

allister-translator committed:
Update co01042103.csv

allister-translator committed:
Update co01042102.csv

allister-translator committed:
Update config.yml
